projects/storefrontlib/src/cms-components/navigation/page-header/page-header.component.ts

```diff
@@ -0,0 +1,34 @@
+import { ChangeDetectionStrategy, Component, OnInit } from '@angular/core';
+import {
+  CmsPageHeaderComponent,
+  PageMeta,
+  PageMetaService,
+} from '@spartacus/core';
+import { Observable } from 'rxjs';
+import { filter, map } from 'rxjs/operators';
+import { CmsComponentData } from '../../../cms-structure/page/model/cms-component-data';
+
+@Component({
+  selector: 'cx-page-header',
+  templateUrl: './page-header.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class PageHeaderComponent implements OnInit {
```

Comment:
Could the component benefit from a more self explanatory name than "page header", since the component doesn't seem to be related to the page header.  

Not saying we should absolutely change the name, but let's discuss.

Reply:
@plabadie what is your proposal? Would the name `PageTitleComponent` give it better?